pyswarms/discrete/binary.py

```diff
@@ -149,7 +149,7 @@ def __init__(
         # Initialize the topology
         self.top = Ring()
 
-    def optimize(self, objective_func, iters, print_step=1, verbose=1):
+    def optimize(self, objective_func, iters, args=(), kwargs={}, print_step=1, verbose=1):
         """Optimizes the swarm for a number of iterations.
```

Comment:
Maybe `*args` and `**kwargs` will work here instead of `args=()` and `kwargs={}`?